packages/tracing/src/browser/browsertracing.ts

```diff
@@ -90,7 +90,7 @@ export interface BrowserTracingOptions extends RequestInstrumentationOptions {
    *
    * Default: undefined
    */
-  _experiments?: Partial<{ enableLongTask: boolean; enableInteractions: boolean }>;
+  _experiments: Partial<{ enableLongTask: boolean; enableInteractions: boolean }>;
```

Comment:
Just to double check here, is this change OK? It is actually always set right now because it is (as `{}`) part of the `DEFAULT_BROWSER_TRACING_OPTIONS`. By actually marking this non-option we can save on checks throughout tracing a bit. cc @AbhiPrasad @Lms24

Reply:
I think this is fine because it's still optional in the integration constructor, and that is all that matters.

Reply:
Jup, this was my thought as well 👍